### PR TITLE
feat(emqx-operator): modified the README and docs files

### PR DESCRIPTION
**Pull request by kiragoo**

### CODE REVIEW

**Review by brianbinbin — APPROVED** [review]

**Review by Rory-Z — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,2 +1,55 @@
 # emqx-operator
+
 A Kubernetes Operator for EMQ X Broker
+
+**Project status: *alpha*** Not all planned features are completed. The API, sepc, status and other user facing objects may chage, but in a backward compatible way.
```
Comment (yiqunc-emqx) [inline README.md:5]: sepc -> spec,  chage -> change

File: README.md [hunk 2]
```diff
@@ -1,2 +1,55 @@
...
+
+Note: Project was previously known as emqx/emqx-operator.
```
Comment (yiqunc-emqx) [inline README.md:7]: 这行可以删掉

File: README.md [hunk 3]
```diff
@@ -1,2 +1,55 @@
...
+
+## Overview
+
+The EMQ X Operator provieds [Kubernetes](https://kubernetes.io/) native deployment and management of [EMQ X](https://www.emqx.io/). The purpose of this project is to simplify and automate the configuration of EMQ X Broker for EMQ X cluster.
```
Comment (yiqunc-emqx) [inline README.md:11]: provieds -> provides

Comment (yiqunc-emqx) [reply README.md:11]: The purpose of this project is to simplify and automate the configuration of EMQ X Broker for EMQ X cluster. ->
The purpose of this project is to simplify and automate the configuration of EMQ X cluster.

File: README.md [hunk 4]
```diff
@@ -1,2 +1,55 @@
...
+
+For an introduction to the EMQ X Operator, see the [getting started](docs/user-guides/getting-started.md) guide.
+
+## Prerequisites
+
+The EMQ X Operator requires a Kubernetes cluster of version `>=1.20.0`.If you are just starting out with the Emqx Operator, it is highly recommended to use the `version:v1.20.0`.
+
+## CustomResourceDefinitions
+
+A core feature of the EMQ X Operator is to monitor the Kubernetes API server for changes to specific objects and ensuer that the current Emqx deployments match thes objects.
```
Comment (yiqunc-emqx) [inline README.md:27]: ensuer -> ensure
the current Emqx deployments match thes objects. -> the running EMQ X deployment matches these objects

File: README.md [hunk 5]
```diff
@@ -1,2 +1,55 @@
...
+## Development
+
+### Prerequisites
+
+- golang environment
+- docker (used for creating container images, etc.)
+- Kubernetes cluster
+  
+## Contributing
+Many files (api, config, controller, hack,...) in this repository are auto-generatoed.
```
Comment (yiqunc-emqx) [inline README.md:47]: auto-generatoed -> auto-generated

File: docs/user-guides/getting-started.md [hunk 6]
```diff
@@ -1,33 +1,30 @@
-# emqx-operator
+**Note**: EMQ X Operator requires use of Kubernetes v1.20.0 and up.
```
Comment (yiqunc-emqx) [inline docs/user-guides/getting-started.md:1]: EMQ X Operator requires use of Kubernetes v1.20.0 and up ->
EMQ X Operator requires Kubernetes v1.20.0 and up

File: docs/user-guides/getting-started.md [hunk 7]
```diff
@@ -1,33 +1,30 @@
...
+### Deployment
 
-### 部署
+* Deploy the operator in the Kubernetes cluster
+* Run the project out of the Kubernetes cluster
 
-* 有两种方式运行 `operator`:
-  * 在 `kubernetes` 集群中以`Deployment`部署`operator`
-  * 在集群外作为`Go`工程项目运行
+>Example: Run the Operator in the ACK service in ALiCloud, the LV and Persistence Volume should prepared before deploying.
```
Comment (yiqunc-emqx) [inline docs/user-guides/getting-started.md:10]: LV 是什么？

File: docs/user-guides/getting-started.md [hunk 8]
```diff
@@ -164,10 +161,10 @@ spec:
...
 ```
 
-> * `cluster` 配置，明细请参考[`cluster`参数配置](https://docs.emqx.cn/enterprise/v4.3/configuration/configuration.html#cluster)
-> * `env` 配置，明细清参考[参数配置](https://docs.emqx.cn/enterprise/v4.3/configuration/configuration.html)
-
-* 确认 `emqx pod` 正常运行
+> * [Details for *cluster* config](https://docs.emqx.io/en/broker/v4.3/configuration/configuration.html)
+> * [Details for *env* config](https://docs.emqx.io/en/broker/v4.3/configuration/configuration.html)
+  
+* Verify the EMQ X pods running
```
Comment (yiqunc-emqx) [inline docs/user-guides/getting-started.md:167]: Verify the EMQ X pods running -> Make sure the EMQ X pods running as expected

File: docs/user-guides/getting-started.md [hunk 9]
```diff
@@ -210,14 +207,13 @@ Start mqtt:wss:external listener on 0.0.0.0:8084 successfully.
...
-> `lb` 及持久化前置依赖资源同上，需提前规划准备部署
-> 注册`crd`资源，同上
+> Prerequirements: LB, Persistence Volume, CRD
 
-* 将 `kubernetes` 集群的配置文件 `.kube/config` 复制到本地 `$HOME/.kube/config`
-* 运行 `main.go` 文件
-* 创建 `cr` 关联的 `RBAC` 及 `cr`
+* Scp the `./kube/config` of Kubernetes cluster to local `$HOME/.kube/config`
+* Run `main.go`
+* Enable the RBAC related to CR and deploy the CR
```
Comment (yiqunc-emqx) [inline docs/user-guides/getting-started.md:216]: Enable the RBAC related to CR and deploy the CR ->
Create RBAC objects from manifest file

File: docs/user-guides/getting-started.md [hunk 10]
```diff
@@ -210,14 +207,13 @@ Start mqtt:wss:external listener on 0.0.0.0:8084 successfully.
...
+2. Running project out of the cluster.
 
-> `lb` 及持久化前置依赖资源同上，需提前规划准备部署
-> 注册`crd`资源，同上
+> Prerequirements: LB, Persistence Volume, CRD
 
-* 将 `kubernetes` 集群的配置文件 `.kube/config` 复制到本地 `$HOME/.kube/config`
-* 运行 `main.go` 文件
-* 创建 `cr` 关联的 `RBAC` 及 `cr`
+* Scp the `./kube/config` of Kubernetes cluster to local `$HOME/.kube/config`
```
Comment (yiqunc-emqx) [inline docs/user-guides/getting-started.md:214]: Make sure kube config is configured properly

File: README.md [hunk 11]
```diff
@@ -1,2 +1,55 @@
...
+
+Note: Project was previously known as emqx/emqx-operator.
+
+## Overview
+
+The EMQ X Operator provieds [Kubernetes](https://kubernetes.io/) native deployment and management of [EMQ X](https://www.emqx.io/). The purpose of this project is to simplify and automate the configuration of EMQ X Broker for EMQ X cluster.
+
+The EMQ X Operator includes, but is not limited to, the following features:
+
+* **Kubernetes Custom Resource**: Use Kubernetes custom resource to deploy and manage EMQ X Broker.
```
Comment (yiqunc-emqx) [inline README.md:15]: Use Kubernetes custom resource to deploy and manage EMQ X Broker. -> Deploy and manage EMQ X Cluster with pre-defined custom resources

File: README.md [hunk 12]
```diff
@@ -1,2 +1,55 @@
...
+
+* **Simplified Deployment Configuration**: Configure the fundamentals of EMQ X like persistence, conf, license from a native Kubernetes resource.
```
Comment (yiqunc-emqx) [inline README.md:17]: Configure the fundamentals of EMQ X like persistence, conf, license from a native Kubernetes resource. ->
Configure the fundamentals of EMQ X Cluster, including persistence, configurations, license and etc, in a Kubernetes-native way.

File: README.md [hunk 13]
```diff
@@ -1,2 +1,55 @@
...
+The EMQ X Operator requires a Kubernetes cluster of version `>=1.20.0`.If you are just starting out with the Emqx Operator, it is highly recommended to use the `version:v1.20.0`.
+
+## CustomResourceDefinitions
+
+A core feature of the EMQ X Operator is to monitor the Kubernetes API server for changes to specific objects and ensuer that the current Emqx deployments match thes objects.
+The Operator acts on the following [custom resource definitions (CRDs)](https://kubernetes.io/docs/tasks/access-kubernetes-api/extend-api-custom-resource-definitions/):
+
+* **`Cluster`**, which defines a desired EMQ X Broker Cluster deployment.
+
+The EMQ X Operator automatically detects changes in the Kubernetes API server to any of the above objects, and ensures that matching deployments and configurations are kept in sync.
```
Comment (yiqunc-emqx) [inline README.md:32]: The EMQ X Operator automatically detects changes in the Kubernetes API server to any of the above objects, and ensures that matching deployments and configurations are kept in sync. ->
The EMQ X Operator automatically detects changes on any of the above custom resource objects, and ensures that running deployments are kept in sync with the changes.

File: docs/user-guides/getting-started.md (hunk 7 above)
Comment (kiragoo) [reply docs/user-guides/getting-started.md:10]: LB，sorry

File: README.md (hunk 2 above)
Comment (kiragoo) [reply README.md:7]: 这行那我就处理掉了

File: README.md (hunk 3 above)
Comment (kiragoo) [reply README.md:11]: Ok

File: README.md (hunk 4 above)
Comment (kiragoo) [reply README.md:27]: Ok

File: README.md (hunk 5 above)
Comment (kiragoo) [reply README.md:47]: tks

**Pull request closed without merging**